functions/examples/template-heredoc-cockroachdb/Makefile

```diff
@@ -1,11 +1,31 @@
 # Copyright 2019 The Kubernetes Authors.
 # SPDX-License-Identifier: Apache-2.0
 
-.PHONY: image
+.PHONY: image, kustomize-digest
+
+FUNCTION_VERSION=v0.1.1
+FUNCTION_IMAGE=gcr.io/kustomize-functions/example-cockroachdb:$(FUNCTION_VERSION)
+
+# Update KUSTOMIZE_IMAGE_DIGEST & FUNCTION_VERSION whenever KUSTOMIZE_VERSION is updated!
+# Use `make kustomize-digest` after changing KUSTOMIZE_VERSION to discover the
+# new value to use for KUSTOMIZE_IMAGE_DIGEST.
+KUSTOMIZE_VERSION=v5.3.0
+KUSTOMIZE_IMAGE_NAME=registry.k8s.io/kustomize/kustomize
+KUSTOMIZE_IMAGE_DIGEST=sha256:74b25b7e6d497336b9f89d152c9770a8318716ea6fe8962791598d195cb11488
```

Comment:
Maybe that is better if we don't set an image digest.
Because digest will change when using different architecture.

Reply:
I only really made this PR to satisfy an over-aggressive scanner that wants all images pinned to digests. If we don't pin a digest, I'll have to configure an exception or downstream patch.

Is the digest really different for different architectures? I thought you could build multi-arch images.

Reply:
I avoided pinning the digest by using go.mod to pin the source checksum instead. Now it builds from source, instead of using the published docker image.